include/grpc/impl/codegen/atm.h

```diff
@@ -19,6 +19,8 @@
 #ifndef GRPC_IMPL_CODEGEN_ATM_H
 #define GRPC_IMPL_CODEGEN_ATM_H
 
+// IWYU pragma: private, include <grpc/impl/codegen/atm.h>
```

Comment:
I'm not sure what this accomplishes, the specified alternative file is the `<...>` way of naming itself. Is this so that IWYU suggests our preferred system-header-style include format?

Reply:
Nah it's got the wrong path... good catch!

Reply:
Fixed now